docs-examples/example-groovy/build.gradle

```diff
@@ -9,6 +10,6 @@ micronaut {
 mainClassName = 'io.micronaut.jms.docs.Application'
 
 project.afterEvaluate {
-    nativeImage.enabled = false
+    nativeCompile.enabled = false
```

Comment:
:x: Avoid using afterEvaluate and referencing tasks directly as it eagerly configures them. Instead you can write:

```
tasks.withType(org.graalvm.buildtools.gradle.tasks.BuildNativeImageTask).configureEach {
    enabled = false
}
```